apis/Google.Cloud.Diagnostics.AspNetCore3/docs/index.md

```diff
@@ -181,7 +177,7 @@ can in inject the `IManagedTracer` into the action method using the `[FromServic
 
 ## Trace Outgoing HTTP Requests (recommended)
 
-The [recommended way of creating HttpClient](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/http-requests?view=aspnetcore-2.1) in ASP.NET Core 2.0 and upwards is to use the
+The [recommended way of creating HttpClient](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/http-requests?view=aspnetcore-3.1) in ASP.NET Core 3.1 and upwards is to use the
```

Comment:
As before - I'd make the link 3.1, but keep it as 2.0 and upwards in the text.

Reply:
Yes, agreed.